Example/Example/Purchase/CheckoutHandler.swift

```diff
@@ -10,19 +10,22 @@ import Afterpay
 import Foundation
 
 final class CheckoutHandler: CheckoutV2Handler {
-
   private let didCommenceCheckoutClosure: () -> Void
   private let onShippingAddressDidChangeClosure: (ShippingAddress) -> Void
+  private let onShippingOptionClosure: (ShippingOption) -> Void
 
   private var checkoutTokenResultCompletion: TokenResultCompletion?
   private var shippingOptionsCompletion: ShippingOptionsCompletion?
+  private var shippingOptionCompletion: ShippingOptionCompletion?
 
   init(
     didCommenceCheckout: @escaping () -> Void,
-    onShippingAddressDidChange: @escaping (ShippingAddress) -> Void
+    onShippingAddressDidChange: @escaping (ShippingAddress) -> Void,
+    onShippingOptionChange: @escaping (ShippingOption) -> Void
```

Comment:
Should this method name include the word "did"? Here and in other places below.

```suggestion
    onShippingOptionDidChange: @escaping (ShippingOption) -> Void
```